Source/relay/TourGuide/Pulls.ash

```diff
@@ -489,13 +489,14 @@ void generatePullList(Checklist [int] checklists)
             pullable_item_list.listAppend(GPItemMake($item[xiblaxian stealth vest], "-combat shirt"));
         if ($item[duonoculars].item_is_usable())
 	        pullable_item_list.listAppend(GPItemMake($item[duonoculars], "-combat, +5 ML"));
-        pullable_item_list.listAppend(GPItemMake($item[ring of conflict], "-combat"));
+        if ($item[ring of conflict].item_is_usable())
```

Comment:
this is not really relevant to this specific PR but i wonder if there is some possible use in eventually disambiguating this to add item_is_usable() more broadly to almost all of this stuff. feels like a good possibility that tptb will keep adding paths that randomly can't use certain shit.